data/version.yaml

```diff
@@ -7,10 +7,10 @@ kubermatic:
       name: v2.13
     - release: v2.12
       name: v2.12
-# kubeone:
-#   name: KubeOne
-#   versions:
-#     - release: master
-#       name: master
-#     - release: v0.12
-#       name: v0.12
+kubeone:
+  name: KubeOne
+  versions:
+    - release: master
+      name: master
+    - release: v0.11
```

Comment:
probably for now we can avoid 0.11 (as we anyway don't have its copy), at new release time we'd start providing 1.0 dedicated docs and onward